### DISCUSSION TITLE
How to not stretch sizer background?

### DISCUSSION THREAD

**Discussion by crapthings** [rexrainbow/phaser3-rex-notes#226] [Q&A]
or should I use a similar size image for the background compared to the sizer?

```
.addBackground(background, { top: 100 })
```

![image](https://user-images.githubusercontent.com/1147704/149164473-d6a78b2a-4c63-44a3-927c-1af6eb6b3fea.png)

![image](https://user-images.githubusercontent.com/1147704/149165768-275486df-8f6b-45cb-bfc5-85e950a0774b.png)

![image](https://user-images.githubusercontent.com/1147704/149165920-8878ef31-ca62-456d-8f17-c68b05b65c8e.png)

or maybe we can stretch horizontally

**Comment by rexrainbow** [comment]
Might try [ninepatch](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ninepatch/) game object.
```javascript
var ninePatch = scene.rexUI.add.ninePatch(x, y, width, height, key, columns, rows, config);
```

**Comment by crapthings** [comment]
@rexrainbow cool

btw the scrollable grid table is awesome
